src/Goteo/Model/Sponsor.php

```diff
@@ -29,12 +28,8 @@ class Sponsor extends \Goteo\Core\Model {
         $name,
         $url,
         $image,
-        $type,
```

Comment:
This field should not be removed. It is used in the Home of Goteo to filter the different types of Sponsors.

We should fix the test instead of removing it from the Model.

Reply:
But there's no 'type' column in the database, neither in the 'goteo' DB nor in 'goteo_test'. Is it a field that is calculated somewhere else?

Reply:
There should be, we added it in this migration -> 20210317154642_goteo_add_sponsor_types.php

Can you check with the console that every migrate has been executed? 

```
docker/exec bin/console migrate status
```

Reply:
Alright, it was very outdated my setup. Even though I had it setup from scratch, by current schema version is: 20201201150349

I'm gonna update it, retest everything again, and reintroduce the type column.

Reply:
Thanks!

Reply:
Just fixed it!!! And the test as well ;)